DataHub/Operation/02_ManageETLProcesses.md

```diff
@@ -0,0 +1,102 @@
+DataHub[!!Omni-Channel](../../Channels/Overview/01_General.md)
+[!!User Interface ETL processes](../UserInterface/02h_ETLProcesses.md)
+[!!User Interface ETL](../UserInterface/02d_ETL.md)
+[!!Manage the ETL mappings](./01_ManageETLMappings.md)
+[!!ETL extensions list](../UserInterface/03_ETLExtensions.md)
+
+
+# Manage the ETL processes
+
+When an ETL attribute mapping has been triggered, the corresponding ETL processes are displayed in the *ETL PROCESSES* tab.
+You can check the status of a current ETL process, confirm a pending ETL process and restart a faulty ETL process.  
+
+
+## Check the ETL process status
+
+After an ETL process has been triggered, the current status is displayed in the *ETL PROCESSES* tab.
+An ETL process can have the following statuses:
+- **Todo**
```

Comment:
To do oder ToDo?
Für mich sieht komisch aus, denn Todo hat eine Bedeutung auf Spanisch...